docs/src/architecture/domain-model.md

```diff
@@ -0,0 +1,121 @@
+# thin-edge Domain Model
+
+The following section introduces the **thin-edge** domain model.
+
+The **thin-edge** domain model explains details of target domains **thin-edge** is primarily designed for.
+It identifies _entities_ and _aspects_ of those target domains, that are touch points for **thin-edge**.
+Additionally it introduces new _entities_ or _aspects_ to seamlessly fit **thin-edge** into those target domains.
+
+Finally the **thin-edge** domain model gives..
+  * _target domain experts_ a clear idea how to position **thin-edge** in their target domain
+  * _target domain experts_ **and** _thin-edge developers_ a common understanding and unique vocabulary for **thin-edge** and it's environment
```

Comment:
That section is intended for the reader. I really like to keep it, since it explains reason and goal for the domain model, and gives a better read flow.

Reply:
I agree this a matter of taste. I prefer texts that simply dive into the main topic without long preambles telling what will be told.